### PR TITLE
Added a validation for a default splunk password

### PR DESCRIPTION
**Pull request by arctan5x**
There is a PR open against the Splunk Docker repo in order to complete this change. We are producing a warning message for an user password that does not conform with the Splunk default password policy.

### CODE REVIEW

File: inventory/environ.py [hunk 1]
```diff
@@ -303,6 +303,13 @@ def prep_for_yaml_out(inventory):
         del inventory_to_dump["splunk"]["shc"]
     return inventory_to_dump
 
+def validate_splunk_default_password(inventory):
+    password = inventory["all"]["vars"]["splunk"]["password"]
+    if password and len(password) >= 8:
```
Comment (outcoldman) [inline inventory/environ.py]: but what if you change this configuration from 8 symbols to 12?

Comment (outcoldman) [reply inventory/environ.py]: I mean `minPasswordLength ` under https://docs.splunk.com/Documentation/Splunk/7.2.0/Admin/Authenticationconf

Comment (lephino) [reply inventory/environ.py]: I think it'd be safe to flag this as a warning that's sent to stdout and not actually stopping the installation?

**Comment by arctan5x** [comment]
any comments?

**Pull request closed without merging**